### PR TITLE
Setup mockbee and created new  backend controllers

### LINKED ISSUE #1: Create backend controllers
## Create:
- Boards controller
- Tasks controllers
- Columns controllers
- Archive tasks controller

### PR DESCRIPTION
**Pull request by kjais1720**
PR for Issue #1 (closes #1)
## Overview
- Setup Mockbee
- Created Backend controllers:
   - `boardsController.js` to handle `get & post` requests to the boards routes and create, update or delete board(s)
   - `tasksController.js` for tasks
   - `columnsController.js` for columns

### CODE REVIEW

File: src/backend/utils/authUtils.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Response } from "miragejs";
+import dayjs from "dayjs";
+import jwt_decode from "jwt-decode";
+
+export const requiresAuth = function (request) {
+  const encodedToken = request.requestHeaders.authorization;
+  const decodedToken = jwt_decode(
+    encodedToken,
+    process.env.REACT_APP_JWT_SECRET
```
Comment (rahulcs754) [inline src/backend/utils/authUtils.js:9]: why you decode token here

**Review by rahulcs754 — APPROVED** [review]
All code look good and clean. 
I think you can remove react image from this project. that will not used in future

File: src/backend/utils/authUtils.js (hunk 1 above)
Comment (kjais1720) [reply src/backend/utils/authUtils.js:9]: This is a backend code, it came with the mockbee setup. It verifies whether the token sent by the app is valid or not

Comment (rahulcs754) [reply src/backend/utils/authUtils.js:9]: okay

File: src/theme.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+const colors = {
+  gray:{
+    800:"hsl(221, 48%, 15%)"
+  },
+  primary:{
+    // light:"hsl(233, 86%, 45%)",
+    light:"rgba(0, 200, 255, 1)",
+    base:"hsl(234, 71%, 25%)",
+    // base:"rgba(0, 97, 215, 1)",
```
Comment (srejitk) [inline src/theme.js:16]: Remove these comments if not needed.

**Review by srejitk — APPROVED** [review]
No changes suggested. Clean code, good luck.

**Pull request merged**